packages/hash/api/src/graphql/resolvers/knowledge/model-mapping.ts

```diff
@@ -0,0 +1,31 @@
+import { EntityModel, PageModel } from "../../../model";
+import { KnowledgeEntity, KnowledgePage } from "../../apiTypes.gen";
+import { entityTypeModelToGQL } from "../ontology/model-mapping";
+
+export type ExternalEntityResolversGQL = "linkedEntities";
+export type UnresolvedEntityGQL = Omit<
+  KnowledgeEntity,
+  ExternalEntityResolversGQL
+>;
+
+export const entityModelToGQL = (
+  entityModel: EntityModel,
+): UnresolvedEntityGQL => ({
+  entityId: entityModel.entityId,
+  entityType: entityTypeModelToGQL(entityModel.entityTypeModel),
+  entityTypeId: entityModel.entityTypeModel.schema.$id,
+  entityVersionId: entityModel.version,
+  ownedById: entityModel.accountId,
```

Comment:
I think moving to `ownedById` wouldn't introduce conflicts for merging `main` into `dev/graph` as all our model classes are separate to the old ones, and any GQL additions we make is of the same nature (additions rather than modification). I think it would be fine to do the renaming in a follow-up for `ontology/*` and `knowledge/*` model classes